NEWS

```diff
@@ -11,7 +11,9 @@ doctest 0.7.0+
 
   * Functions defined directly in the Octave interpreter can now be tested.
 
-  * Test suite fixes for upcoming Octave 6.
+  * Test suite fixes for Octave 6.
+
+  * Source code is in encoded UTF-8, indicated with `.oct-config` files.
```

Comment:
I'm not a native speaker. But the grammar in that sentence sounds a bit odd to me.